runners/flink/src/test/java/org/apache/beam/runners/flink/PortableTimersExecutionTest.java

```diff
@@ -157,20 +157,13 @@ public void onTimer(
           }
         };
 
-    DoFn<KV<String, Integer>, Void> collectResults =
-        new DoFn<KV<String, Integer>, Void>() {
-          @ProcessElement
-          public void processElement(ProcessContext context) {
-            results.add(context.element());
-          }
-        };
-
     final Pipeline pipeline = Pipeline.create(options);
-    pipeline
-        .apply(Impulse.create())
-        .apply(ParDo.of(inputFn))
-        .apply(ParDo.of(testFn))
-        .apply(ParDo.of(collectResults));
+    PCollection<KV<String, Integer>> output =
+        pipeline.apply(Impulse.create()).apply(ParDo.of(inputFn)).apply(ParDo.of(testFn));
+    PAssert.that(output).containsInAnyOrder(expectedOutput);
```

Comment:
+1 to use PAssert to verify result of a Pipeline.

Reply:
+1